splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -327,12 +330,126 @@ def _get_python_version_from_executable(python_binary_name: str) -> str:
         )
 
 
+def summary_report(
+    source: str,
+    ta_name: str,
+    config_path: Optional[str] = None,
+    output_directory: Optional[str] = None,
+    verbose_report: Optional[str] = None,
+):
+    # initialising colorama to handle ASCI color in windows cmd
+    c.init()
+    color_pallete = {
+        "copied": c.Fore.GREEN,
+        "conflict": c.Fore.RED,
+        "modified": c.Fore.YELLOW,
+    }
+
+    # conflicting files from ucc-gen package folder
+    conflict_path = os.path.join(internal_root_dir, "package")
+    # conflict files generated through-out the process
+    conflict_static_list = [
+        "import_declare_test.py",
+        f"{ta_name}_rh_*.py",
+        "app.conf",
+        "inputs.conf*",
+        "restmap.conf",
+        "server.conf",
+        f"{ta_name}_*.conf*",
+        "web.conf",
+        "default.xml",
+        "configuration.xml",
+        "dashboard.xml",
+        "inputs.xml",
+        "openapi.json",
+    ]
+
```

Comment:
> @[artemrys](https://github.com/artemrys)(https://github.com/splunk/addonfactory-ucc-generator/pull/918/files#r1368511362)
> Can we use the full path to the files instead of trying to match with regex?

It is a only a part of file check and based on the outcome it will decide what type of issue the user might have.
The process is as follows:
1. Checks if file exists at same relative path (from output) in package (so the paths are checked here)
2. Then checks if the file is flagged as conflict generating file (thus the regex list)
3. If no conflict script checks if the file was modified
4. If not modified - marks as copy
5. Otherwise just gives the created mark